client/trino-jdbc/pom.xml

```diff
@@ -367,6 +367,21 @@
         </pluginManagement>
 
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>nonshaded</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>nonshaded</classifier>
```

Comment:
is it worth a code comment why this is useful?

Reply:
Sure. I can add a comment to the commit and in the pom.